### ISSUE TITLE
Contextual menu items are navigable by Tab as well as arrow keys. Should be only arrow keys.

### ISSUE THREAD

**Issue by HarshUpadhyay-MSFT** [microsoft/fluentui#16909]
### Environment Information

- **Package version(s)**: 7.117.0
- **Browser and OS versions**: Google Chrome Version 88.0.4324.150 (Official Build) (64-bit), Windows 10 Enterprise version 20H2

### Describe the issue:

Once the contextual menu is expanded, we can use both arrow as well as tab keys to cycle through menu items. It should just be arrow keys. Having both can confuse users.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/HarshUpadhyay-MSFT/pen/qBqaRpX

#### Actual behavior:

Both Tab and arrow keys can cycle through menu items 

#### Expected behavior:

Only arrow keys should cycle through menu items

### Documentation describing expected behavior

If both Tab and arrow keys can cycle through menu items it can confuse the user.

**Comment by HarshUpadhyay-MSFT** [comment]
reported internally by Accessibility testers

**Comment by joschect** [comment]
@HarshUpadhyay-MSFT Thanks for the submission. This is intentional behavior. However, we understand that not everyone wants it to act that way.  You can disable it by passing in ` focusZoneProps: {handleTabKey: FocusZoneTabbableElements.none}` to the contextual menu. I've adapted your [codepen to demonstrate this behavior](https://codepen.io/slimemold/pen/QWGxNLW?editors=0011) . Let me know if you have any other questions!
